### PR TITLE
Fix wording on profile page

### PR DESCRIPTION
**Pull request by Pytal**
See title 👆

### CODE REVIEW

File: core/src/views/Profile.vue [hunk 1]
```diff
@@ -255,6 +255,12 @@ export default {
 			// For some reason the returned string has prepended whitespace
 			return getComputedStyle(document.body).getPropertyValue('--color-main-background').trim()
 		},
+
+		emptyProfileMessage() {
+			return this.isCurrentUser
+				? t('core', 'You haven\'t added any info yet')
+				: (this.displayname || this.userId) + ' ' + t('core', 'hasn\'t added any info yet')
```
Comment (juliushaertl) [inline core/src/views/Profile.vue:262]: Building sentences like that might not work in all languages, better include the name as a parameter in the translatable text.
```suggestion
				:  + t('core', '{user} hasn\'t added any info yet', { user: (this.displayname || this.userId) })
```

**Review by juliusknorr — CHANGES_REQUESTED** [review]

File: core/src/views/Profile.vue (hunk 1 above)
Comment (Pytal) [reply core/src/views/Profile.vue]: Done!

**Review by juliusknorr — APPROVED** [review]

**Review by artonge — APPROVED** [review]

**Pull request merged**